tasks/change-granule-collection-s3/tests/test-index.js

```diff
@@ -0,0 +1,1000 @@
+'use strict';
```

Comment:
The tests as exist look good, but there are enough methods abstracted we should probably more directly test them (happy path concerns) beyond 'did the files move', particularly if we're asserting just that an object exists and the like.